packages/data-context/src/sources/migration/codegen.ts

```diff
@@ -202,18 +206,33 @@ function formatObjectForConfig (obj: Record<string, unknown>) {
   return JSON.stringify(obj, null, 2).replace(/^[{]|[}]$/g, '') // remove opening and closing {}
 }
 
-function createE2eTemplate (pluginPath: string, hasPluginsFile: boolean, options: Record<string, unknown>) {
-  const requirePlugins = `return require('./${pluginPath}')(on, config)`
```

Comment:
Removed this `./`, I don't think we should be adding this - just receive + interpolate their `pluginsFile` as is. Eg if they did

```json
{
  "pluginsFile": "./plugins.js"
}
```

And we interpolate, we'd end up with `'././plugins.js`, which is weird, even if might technically work.